cl_dll/saytext.cpp

```diff
@@ -221,6 +221,14 @@ void CHudSayText :: SayTextPrint( const char *pszBuf, int iBufSize, int clientIn
 		ConsolePrint( pszBuf );
 		return;
 	}
+	char timestamp[16];
+	std::time_t curtime = std::time(nullptr);
+	auto written = std::strftime(timestamp, sizeof(timestamp), "[%H:%M:%S] ", std::localtime(&curtime));
+
+	if(written > 0)
+	    ConsolePrint( timestamp );
+
+	ConsolePrint( pszBuf );
```

Comment:
I see there's a `convert_locations` call below that should probably happen before the console print, maybe something else too?

Reply:
![image](https://user-images.githubusercontent.com/5108747/118871641-09b96300-b8e8-11eb-8bbe-afc1b049c7f5.png)

Should be fine now.